ui/dashboards/src/components/Panel/Panel.tsx

```diff
@@ -37,11 +38,18 @@ export function Panel(props: PanelProps) {
     return { width, height };
   }, [width, height]);
 
+  const { ref, inView } = useInView({
+    threshold: 0,
+    initialInView: true,
+    triggerOnce: true,
```

Comment:
Ahh nice. Does this option mean that once it's in view and rendered, it doesn't unmount (and then get re-rendered later)?

Reply:
Yep that's right! Don't want to rerun a bunch of queries if we don't have to